src/app.js

```diff
@@ -111,56 +57,18 @@ app.on('ready', () => {
       { label: `Best Streak: ${bestStreak}`, enabled: false },
       { label: `Contributions: ${contributionCount}`, enabled: false },
       { type: 'separator' },
-      { label: 'Reload', accelerator: 'Cmd+R', click: requestContributionData },
+      { label: 'Reload', accelerator: 'CmdOrCtrl+R', click: requestContributionData },
       {
         label: 'Open GitHub Profile...',
-        accelerator: 'Cmd+O',
+        accelerator: 'CmdOrCtrl+O',
         click: () => shell.openExternal(githubProfileUrl)
       },
       { type: 'separator' },
       {
-        label: 'Set GitHub Username...',
-        accelerator: 'Cmd+S',
-        click: createUsernameWindow
+        label: 'Preferences...',
+        accelerator: 'CmdOrCtrl+,',
+        click: createPreferencesWindow
```

Comment:
We shouldn't be calling `createPreferencesWindow` directly here.

Instead this should called something like `onPreferencesClick` which checks to see if it already exists, if it does it should be focused, if it doesn't then it should be created.